site/en/docs/extensions/mv3/webhid/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: "layouts/doc-post.njk"
+title: "WebHID in extensions"
+seoTitle: "Chrome Extensions: WebHID"
+date: 2023-11-02
+description: The WebHID API, which exposes Human Interface Device (HID) compatible devices to the web, is available in extensions.
+
+---
+
+A Human Interface Device (HID) takes input from or provides output to humans. It also refers to the HID protocol, a standard for bi-directional communication between a host and a device that is designed to simplify the installation procedure.
+
+This page describes aspects of the API that are particular to extensions. Refer to MDN for complete details of the [WebHID API](https://developer.mozilla.org/docs/Web/API/WebHID_API).
+
+You can find a [sample app for WebHID](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/sample.co2meter) in our samples repo.
+
+## Availability in extensions
+
+Chrome 117 or later.
+
+## Permissions
+
+No manifest file permissions are required; however WebHID triggers the browser's user permission flow.
+
+## Manifest
+
+No manifest keys are needed for this API.
+
+## Supporting contexts
+
+This API may be used in any context. The `WebHID.requestDevice()` method cannot be used in extension service workers. See the next section for details.
```

Comment:
This could be more clear since later it says you can use it on the service worker via messaging. 
Could we rephrase it like it cannot be used directly in a sw, but there is a workaround in the next section?